staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_test.go

```diff
@@ -132,6 +135,32 @@ func newTestWatchCache(capacity int, indexers *cache.Indexers) *testWatchCache {
 	return wc
 }
 
+type alwaysFiringTicker struct {
+	stopCh chan struct{}
+}
+
+func (t *alwaysFiringTicker) NewTicker(d time.Duration) clock.Ticker {
+	return t
```

Comment:
This seems strange...
If I create two tickers and then close one of those, it will also close the other. Can you please make returned tickers self-contained?

Reply:
Done